src/query/v2/interpreter.hpp

```diff
@@ -17,6 +17,7 @@
 #include "coordinator/coordinator.hpp"
 #include "coordinator/coordinator_client.hpp"
 #include "io/local_transport/local_transport.hpp"
+#include "io/simulator/simulator_transport.hpp"
```

Comment:
I think these two headers can be removed as the interpreter should not depend on any of them.